scripts/ci/debian/control

```diff
@@ -0,0 +1,15 @@
+Source: mono-llvm
+Priority: optional
+Maintainer: Alex Kyte <alkyte@microsoft.com>
+Build-Depends: debhelper (>= 8.0.0)
+Standards-Version: 3.9.4
+Section: libs
+Homepage: https://github.com/mono/llvm.git
+#Vcs-Git: git://git.debian.org/collab-maint/mono-llvm.git
+#Vcs-Browser: http://git.debian.org/?p=collab-maint/mono-llvm.git;a=summary
+
+Package: mono-llvm
+Section: libs
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: The mono fork of llvm
```

Comment:
You can give a more detailed description in the space below the Description line if you like (lines must start with a space, use ` .` for an empty line break).

Descriptions are useful, as they are used in package searches.